common/src/main/java/org/apache/flink/training/exercises/common/datatypes/TaxiRide.java

```diff
@@ -139,7 +132,7 @@ public int compareTo(@Nullable TaxiRide other) {
         if (other == null) {
             return 1;
         }
-        int compareTimes = Long.compare(this.getEventTime(), other.getEventTime());
+        int compareTimes = Long.compare(this.getEventTimeMillis(), other.getEventTimeMillis());
```

Comment:
Why not use `java.time.Instant#compareTo`?
```suggestion
        int compareTimes = this.eventTime.compareTo(other.eventTime);
```